mta/tests/migration/controls/test_stakeholder.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_email, generate_string
+
+
+@pytest.mark.stakeholder
+def test_stakeholder_crud(stakeholder, stakeholder_groups, get_api, delete_api, update_api):
```

Comment:
Please remove `stakeholder_groups` from the func definition since it is not used

Reply:
Done